packages/material-ui-lab/src/internal/pickers/PickersArrowSwitcher.tsx

```diff
@@ -61,10 +61,10 @@ export const styles: MuiStyles<PickersArrowSwitcherClassKey> = (
   },
 });
 
-const PickersArrowSwitcher = React.forwardRef<
-  HTMLDivElement,
-  ArrowSwitcherProps & WithStyles<typeof styles>
->((props, ref) => {
+const PickersArrowSwitcher = React.forwardRef(function PickersArrowSwitcher(
+  props: ArrowSwitcherProps & WithStyles<typeof styles>,
+  ref: React.Ref<HTMLDivElement>,
+) {
```

Comment:
Reasons for 

```diff
-React.forwardRef<RefType, PropType>(function Component(props, ref)
+React.forwardReffunction Component(props: PropType, ref: React.Ref<RefType>)
```

We have to move the types if we drop `React.forwardRef` with the original code. We don't have to move the types in the second case since they're already declared where they're used. 

Not declaring types where they're used makes sense when they can be inferred from the context. But that is not the case here.